partners/socialLogins.md

```diff
@@ -22,5 +22,5 @@ We've rounded up the best social login options out there, all in one spot for yo
 | [**Privy**](/Account/signers/privy)          | Choose between having popups for more transparency vs abstract signing away | [Authentication Methods](https://docs.privy.io/guide/configuration/#login-methods)                      | SSS                                                                                                                                                                   |
 | [**Capsule**](/Account/signers/capsule)        | Policy engine for sessions                                                  | [Demo Apps](https://docs.usecapsule.com/getting-started/examples)                                       | MPC + DKG ([Link](https://docs.usecapsule.com/how-capsule-works/key-management))                                                                                      |
 | [**Magic**](/Account/signers/magic)         |                                                                             | [Authentication Methods](https://magic.link/docs/authentication/overview)                               | Delegated Key Management System ([DKMS](https://magic.link/docs/home/security/product-security#private-keys-can-be-lost-or-stolen-how-do-you-protect-my-private-key)) |
-| **Lit Protocol** | Programmable key pairs                                                      | [More about PKPs](https://developer.litprotocol.com/v2/pkp/intro/)                                      | MPC                                                                                                                                                                   |
+| **Lit Protocol** | Programmable key pairs                                                      | [More about PKPs]                                      | MPC                                                                                                                                                                   |
```

Comment:
@Khushalithakkar1993 could you please add the correct URLs here. And Please create a single PR for all these changes.